### PR TITLE
Port TRAC-IK tutorial to ROS 2

### PR DESCRIPTION
**Pull request by sea-bass**
This PR ports the TRAC-IK tutorial to ROS 2 now that it's available.

Closes https://github.com/ros-planning/moveit2_tutorials/issues/28

### CODE REVIEW

File: doc/how_to_guides/trac_ik/trac_ik_tutorial.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+Install
+-------
+
+The ``rolling-devel`` branch of the TRAC-IK repository has the latest ROS 2 implementation.
+For now, the repository must be built from source in your ROS 2 workspace, for example ``~/moveit2_ws``.
+
+```
+cd ~/moveit2_ws/src
+git clone -b rolling-devel https://bitbucket.org/traclabs/trac_ik.git
+```
```
Comment (marioprats) [inline doc/how_to_guides/trac_ik/trac_ik_tutorial.rst:27]: This code block doesn't render correctly for me in rich diff mode, but syntax looks correct?

File: doc/how_to_guides/trac_ik/trac_ik_tutorial.rst [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+Usage
+-----
+
+- Find the MoveIt :doc:`kinematics.yaml </doc/examples/kinematics_configuration/kinematics_configuration_tutorial>` file created for your robot.
```
Comment (marioprats) [inline doc/how_to_guides/trac_ik/trac_ik_tutorial.rst:32]: same here, this doesn't render correctly to me.

Comment (sea-bass) [reply doc/how_to_guides/trac_ik/trac_ik_tutorial.rst:32]: I built the HTML locally and this one shows up fine.

image

File: doc/how_to_guides/trac_ik/trac_ik_tutorial.rst (hunk 1 above)
Comment (sea-bass) [reply doc/how_to_guides/trac_ik/trac_ik_tutorial.rst:27]: Built this one locally and it *does not* show up fine. Will fix:

image

Comment (sea-bass) [reply doc/how_to_guides/trac_ik/trac_ik_tutorial.rst]: updated to

image

**Review by marioprats — APPROVED** [review]

**Pull request merged**